packages/webview_flutter/lib/webview_flutter.dart

```diff
@@ -397,6 +402,45 @@ class WebView extends StatefulWidget {
   /// By default `userAgent` is null.
   final String? userAgent;
 
+  /// (Android only) Sets whether the WebView should support zooming using its on-screen zoom controls and gestures.
+  ///
+  /// The particular zoom mechanisms that should be used can be set with setBuiltInZoomControls(boolean).
+  ///
+  /// By default 'setSupportZoom` is true
+  final bool setSupportZoom;
+
+  /// (Android only) Sets whether the WebView should use its built-in zoom mechanisms.
+  ///
+  /// The built-in zoom mechanisms comprise on-screen zoom controls, which are displayed over the WebView's content, and the use of a pinch gesture to control zooming.
+  ///
+  /// By default 'setBuiltInZoomControls` is true
+  final bool setBuiltInZoomControls;
+
+  /// (Android only) Sets whether the WebView should use its built-in zoom mechanisms.
+  ///
+  /// The built-in zoom mechanisms comprise on-screen zoom controls, which are displayed over the WebView's content, and the use of a pinch gesture to control zooming.
+  ///
+  /// By default 'setDisplayZoomControls` is false
+  final bool setDisplayZoomControls;
+
+  /// (Android only) Sets whether the WebView should enable support for the "viewport" HTML meta tag or should use a wide viewport.
+  ///
+  /// When the value of the setting is false, the layout width is always set to the width of the WebView control in device-independent (CSS) pixels.
+  ///
+  /// When the value is true and the page contains the viewport meta tag, the value of the width specified in the tag is used.
+  ///
+  /// If the page does not contain the tag or does not provide a width, then a wide viewport will be used.
+  ///
+  /// By default 'setUseWideViewPort` is true
+  final bool setUseWideViewPort;
```

Comment:
Am I correct to understand that WkWebView essentially always keeps the viewport tag enabled, and programmatically tweaking the viewport is done with JS evaluation? (e.g what's discussed on [this SO thread]( https://stackoverflow.com/questions/26295277/wkwebview-equivalent-for-uiwebviews-scalespagetofit)).

If that is the case it sounds like we can end up with a consistent API without losing flexibility by just always setting UseWideViewPort on Android?

Reply:
It does look like [WKWebView](https://developer.apple.com/documentation/webkit/wkwebview) defaults to keeping the viewport tag enabled, but I think the API still allows for some simple controlling of the scale and zoom levels to where it does seem like a consistent API would be possible.

From WKWebView docs 

`var pageZoom: CGFloat
`The scale factor by which the web view scales content relative to its bounds.

`var allowsMagnification: Bool
`A Boolean value that indicates whether magnify gestures change the web view’s magnification.

`var magnification: CGFloat
`The factor by which the page content is currently scaled.

`func setMagnification(CGFloat, centeredAt: CGPoint)
`Scales the page content and centers the result on the specified point.

https://github.com/flutter/flutter/issues/75196 was filed to address this


No code changes were made as a result of this comment so please advise if you intended me to make a change to this PR